src/modules/rover_pos_control/RoverPositionControl.hpp

```diff
@@ -113,15 +114,15 @@ class RoverPositionControl : public ModuleBase<RoverPositionControl>, public Mod
 
 	uORB::Subscription	_parameter_update_sub{ORB_ID(parameter_update)};
 
-	manual_control_setpoint_s		_manual{};			/**< r/c channel data */
+	manual_control_setpoint_s		_manual{};			    /**< r/c channel data */
 	position_setpoint_triplet_s		_pos_sp_triplet{};		/**< triplet of mission items */
 	vehicle_attitude_setpoint_s		_att_sp{};			/**< attitude setpoint > */
 	vehicle_control_mode_s			_control_mode{};		/**< control mode */
 	vehicle_global_position_s		_global_pos{};			/**< global vehicle position */
 	vehicle_local_position_s		_local_pos{};			/**< global vehicle position */
-	actuator_controls_s			_act_controls{};		/**< direct control of actuators */
-	vehicle_attitude_s			_vehicle_att{};
-	sensor_combined_s			_sensor_combined{};
+	actuator_controls_s			    _act_controls{};		/**< direct control of actuators */
+	vehicle_attitude_s              _vehicle_att{};
```

Comment:
Please fix the indentation errors

Reply:
OK, will do. 
I run the fix_code_style.sh, thought that took care of that sort of thing. But just realized that CLion was using tabs as spaces, will sort it out.

Reply:
Thanks!